modules/hub/src/OnchainTransactionService.ts

```diff
@@ -301,10 +302,26 @@ export class OnchainTransactionService {
         return
       }
 
-      if (!tx || !tx.blockNumber) {
-        const txnAgeS = (Date.now() - (+new Date(txn.submittedOn))) / 1000
-        const txnAge = `${Math.floor(txnAgeS / 60)}m ${Math.floor(txnAgeS % 60)}s`
+      const txnAgeS = (Date.now() - (+new Date(txn.submittedOn))) / 1000
+      const txnAge = `${Math.floor(txnAgeS / 60)}m ${Math.floor(txnAgeS % 60)}s`
+
+      // not in mempool yet
+      if (!tx) {
+        // resubmit after 60 seconds of waiting
+        // TODO: shorten this?
+        if (txnAgeS > 60) {
+          // sign and send, dont change status here
+          // should reset the submit time
+          await this.signAndSendTx(txn)
```

Comment:
Per your above comment, we would never get here:
> Doesn't handle the case where tx is null (possibly just changing the if-statement below so it's if (!tx || err) or similar would be sufficient)

I didn't add the above if statement because of this, but if you think it's better to delete this block and just retry on the next polling loop, I'm okay with that too.